### PR TITLE
chore: run tests on Windows

### PR DESCRIPTION
**Pull request by kyegupov**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "eslint": "eslint -c .eslintrc src",
     "tslint": "tslint --project tsconfig.json --format stylish --exclude **/src/**/*.js",
     "prepare": "npm run build",
-    "tap": "tap test/*.test.* -Rspec --timeout=300 --node-path ts-node --test-file-pattern '/\\.[tj]s$/'",
+    "tap": "tap test/*.test.* -Rspec --timeout=300 --node-arg=-r --node-arg=ts-node/register --test-file-pattern=/[.][tj]s$/",
```
Comment (gjvis) [inline package.json]: lets rather just drop the single-quotes – no need for additional change in this PR?

Comment (kyegupov) [reply package.json]: Is node-path still reliable? Then why not.

Comment (gjvis) [reply package.json]: node-path is not reliable, I was just referring to the test-file-pattern (sorry, that wasn't clear!)

force pushed a change

**Comment by kyegupov** [comment]
Weird build error in AppVeyor. Let's close and reopen and see if it was a fluke.

**Comment by kyegupov** [comment]
Kicking the flaky Appveyor build

**Comment by kyegupov** [comment]
Kicking the build again to verify if the errors are flakiness or legit

**Pull request merged**